be/Cf.WebApi/Endpoints/ServiceDocumentEndpoints.cs

```diff
@@ -17,44 +19,25 @@ public static void MapServiceDocumentRoutes(this IEndpointRouteBuilder builder)
             .WithTags(Tag)
         .HasApiVersion(1);
 
-        group.MapPost("UploadPdf", UploadPdfAsync);
         group.MapPost("UploadpdfBytes", UploadPdfBytesAsync);
     }
 
     [Authorize(Roles = "Service")]
-    private static async Task UploadPdfAsync([FromServices] IServiceDocumentService pdfService, IHttpContextAccessor httpContextAccessor, [FromForm] IFormFile file)
+    private static async Task UploadPdfBytesAsync([FromServices] IServiceDocumentService pdfService, IHttpContextAccessor httpContextAccessor, [FromBody] string base64Content)
     {
-        if (file == null || file.Length == 0)
+        if (string.IsNullOrEmpty(base64Content))
```

Comment:
It is not a good practice to handle exceptions in the endpoint. Lets move this logic to `SavePdfAsync` method